docs/content/integrations/dbt.mdx

```diff
@@ -1,380 +1,159 @@
 
-If you know what models you want to select ahead of time, you might prefer specifying this while configuring your resource. Because you aren't specifying any specific arguments at runtime, you can use the prebuilt `dbt_run_op`, instead of writing your own.
+```python
+import json
+from dagster_dbt import load_assets_from_dbt_manifest
 
-```python file=/integrations/dbt.py startafter=start_marker_dbt_cli_run_specific_models_preconfig endbefore=end_marker_dbt_cli_run_specific_models_preconfig dedent=4
-from dagster_dbt import dbt_cli_resource, dbt_run_op
-
-from dagster import job
-
-my_dbt_resource = dbt_cli_resource.configured(
-    {"project_dir": "path/to/dbt/project", "models": ["tag:staging"]}
+dbt_assets = load_assets_from_dbt_manifest(
+    json.load("path/to/dbt/manifest.json", encoding="utf8"),
 )
-
-@job(resource_defs={"dbt": my_dbt_resource})
-def my_dbt_job():
-    dbt_run_op()
 ```
 
-##### **... supplying an argument**
+Note: if you make any changes to your dbt project that change the structure of the project (such as changing the dependencies of a model or adding a new one), you'll need to regenerate your manifest file for those changes to be reflected in Dagster.
 
-If you want to change which models you select depending on what happens during execution, you should create your own op that invokes the dbt resource (rather than using a prebuilt op). This allows you to explicitly set the `models` parameter at runtime. This would look similar to the following:
+## Adding a resource
 
-```python file=/integrations/dbt.py startafter=start_marker_dbt_cli_run_specific_models_runtime endbefore=end_marker_dbt_cli_run_specific_models_runtime dedent=4
-from dagster import op
+In order to function, these assets require a "dbt" resource, which will be responsible for firing off dbt cli commands. The `dagster-dbt` integration provides the `dbt_cli_resource` to make this easy. This resource can be configured with CLI flags that will be passed i
 
-@op(required_resource_keys={"dbt"})
-def run_models(context, some_condition: bool):
-    if some_condition:
-        context.resources.dbt.run(models=["tag:staging"])
-    else:
-        context.resources.dbt.run(models=["tag:other"])
-```
-
-#### Using a different dbt profile for different dagster modes
-
-Dagster supports creating [multiple jobs from the same graph](/concepts/ops-jobs-graphs/jobs#from-a-graph). dbt has a similar concept, [profiles](https://docs.getdbt.com/dbt-cli/configure-your-profile). You might want to run a dev version of your graph that targets the development-specific dbt profile, and then have a prod version that runs using the prod dbt profile. This example shows how to accomplish this.
-
-```python file=/integrations/dbt.py startafter=start_marker_dbt_cli_profile_modes endbefore=end_marker_dbt_cli_profile_modes dedent=4
-from dagster_dbt import dbt_cli_resource, dbt_run_op
+You can configure this resource and add it to your dbt assets:
```

Comment:
This comes across as an incomplete sentence - maybe just adding `by` to the end before the colon would help. That links the copy to the example that follows.